modules/ROOT/pages/deploy-on-premises.adoc

```diff
@@ -0,0 +1,195 @@
+= Deploy Applications to On-Premises Mule Instances
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can deploy Mule Applications to your On-Premises Mule Instances using any of the following tools:
```

Comment:
GLOBALLY change Mule Applications to "Mule applications" and also lowercase "on-premises Mule instances."

Reply:
Also...would it be a good idea to add something about under what circumstances your audience wants or needs to do perform this action in this way, maybe as opposed to other ways?